terraform/modules/tf_threat_intel_downloader/dynamodb.tf

```diff
@@ -18,3 +18,80 @@ resource "aws_dynamodb_table" "threat_intel_ioc" {
     Name = "StreamAlert"
   }
 }
+
+// IAM Role: Application autoscalling role
+resource "aws_iam_role" "appautoscaling" {
+  name               = "${var.prefix}_streamalert_appautoscaling"
+  assume_role_policy = "${data.aws_iam_policy_document.appautoscaling_assume_role_policy.json}"
+}
+
+// IAM Policy Doc: Generic Application Autoscaling AssumeRole
+data "aws_iam_policy_document" "appautoscaling_assume_role_policy" {
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["application-autoscaling.amazonaws.com"]
+    }
+  }
+}
+
+// IAM Role Policy: Allow appautoscaling IAM role to autoscaling DynamoDB table
+resource "aws_iam_role_policy" "appautoscaling" {
+  name   = "DynamoDBAppAutoscalePolicy"
+  role   = "${aws_iam_role.appautoscaling.id}"
+  policy = "${data.aws_iam_policy_document.appautoscaling.json}"
+}
+
+// IAM Policy Doc: Allow autoscaling IAM role to send alarm to CloudWatch
+// and change table settings for autoscaling.
+data "aws_iam_policy_document" "appautoscaling" {
+  statement {
```

Comment:
respective `dynamodb` and `cloudwatch` permissions should be split into two statements